include/rocksdb/statistics.h

```diff
@@ -357,10 +357,15 @@ enum Tickers : uint32_t {
   FILES_DELETED_IMMEDIATELY,
 
   // The counters for error handler, not that, bg_io_error is the subset of
-  // bg_error and bg_retryable_io_error is the subset of bg_io_error
+  // bg_error and bg_retryable_io_error is the subset of bg_io_error.
+  // The misspelled versions are deprecated and will be removed in a future
+  // release.
   ERROR_HANDLER_BG_ERROR_COUNT,
+  ERROR_HANDLER_BG_ERROR_COUNT_MISSPELLED,
```

Comment:
nit: would you mind adding this note as a such as TODO : deprecate the misspelled tickers in next major release.